src/flow_solver/flow_solver.cpp

```diff
@@ -457,6 +483,14 @@ int FlowSolver<dim,nstate>::run() const
             ode_solver->initialize_steady_polynomial_ramping(poly_degree);
         }
         ode_solver->steady_state();
+        
+        bool use_mesh_adaptation = (all_param.mesh_adaptation_param.total_mesh_adaptation_cycles > 0);
+        
+        if(use_mesh_adaptation)
+        {
+            perform_mesh_adaptation();
+        }
+
         flow_solver_case->steady_state_postprocessing(dg);
```

Comment:
this would have to also be done inside `perform_mesh_adaptation` so that lift/drag values are being outputted throughout the refinement cycles

Reply:
Yeah, that would make it convenient to view functional values. Fixed in the latest commit.